### ISSUE TITLE
Prevent empty space messages

### ISSUE THREAD

**Issue by yazeedb** [pusher/react-slack-clone#71]
Hi,

It seems users can currently send empty spaces as messages.
Would a PR patching this be helpful?

![screen shot 2018-11-30 at 7 49 34 am](https://user-images.githubusercontent.com/6687119/49290402-e7f86700-f474-11e8-8e58-1c8825917315.png)

![screen shot 2018-11-30 at 7 51 01 am](https://user-images.githubusercontent.com/6687119/49290455-052d3580-f475-11e8-9174-7f1788721a58.png)

**Comment by lukejacksonn** [comment]
As far as the Chatkit API is concerned.. this is a valid message

**Comment by yazeedb** [comment]
True. But should this app's UX conform to an API spec, or the app being modeled after, Slack?

Just found empty space messages jarring as a user, reminds me of Microsoft Lync 😨

**Comment by mike1136** [comment]
Some input validation should fix this. Might give it a try later

**Comment by lukejacksonn** [comment]
Yes, this should not be too much work to add. @yazeedb makes a good point about the UX. WOuld be happy to look at a PR for it!
